ru/mail/chrome/messenger/messengercompose/intlKeysHandlerOverlay.dtd

```diff
@@ -1,4 +1,4 @@
 <!-- Intl Chars -->
-<!ENTITY moveToNextPage.label "Show next page of characters">
-<!ENTITY moveToPreviousPage.label "Show previous page of characters">
+<!ENTITY moveToNextPage.label "�������� ��������� �������� ��������">
+<!ENTITY moveToPreviousPage.label "�������� ���������� �������� ��������">
```

Comment:
This is outdated file, it was in the ANSI encoding and therefore you see the garbage cahrs. The correct encoding is UTF-8 and I have fixed it on Aug 22nd. Please use the latest corrected file.